### ISSUE TITLE
Pressing escape does nto dismiss read marker for threads

### ISSUE THREAD

**Issue by davidegirardi** [vector-im/element-web#25437]
### Steps to reproduce

1. Open a room with one or more unread threads
2. Notice the dot near the thread icon indicating there is an unread thread
3. Press escape

#### What did you expect?
All the threads are marked as read since pressing escape dismisses the read marker.

#### What happened instead?
The read marker disappears but the threads are still marked as unread.

### Operating system

Arch Linux

### Browser information

Firefox 113.0.2

### URL for webapp

develop.element.io/

### Application version

Element version: 0a5173f25802-react-2571f54e8b2b-js-473209873131 Olm version: 3.2.14

### Will you send logs?

No

**Comment by t3chguy** [comment]
This isn't the currently expected behaviour based on the keyboard help page

<img width="646" alt="image" src="https://github.com/vector-im/element-web/assets/2403652/dc604d0e-cc9a-4cb7-ba71-c9126261ff9b">

Read markers only exist in the main timeline (m.fully_read) - do you mean read receipts? Read markers aren't something that supports threads https://spec.matrix.org/v1.6/client-server-api/#mfully_read

**Comment by davidegirardi** [comment]
No, I mean read markers, the thing you jump to when pressing Shift+PageUp. I thought that dismissing that removed the notification dot from the room name, but I just noticed that it happens as soon as I click on the room name 🤔

Is that new or has it always been like that?

**Comment by t3chguy** [comment]
Unreads are based on read receipts, that's what the spec/clients/servers use to mark something as read, `m.fully_read` (read markers, green horizontal rule) are just visual markers to inform users the oldest message they've not yet read, RRs instead track the latest seen event (which is what informs what is unread)

**Comment by davidegirardi** [comment]
Gotcha. I think I need to open a new issue and ask for a shortcut for "Mark as read" instead of right clicking on the room name in the sidebar then, unless there's already one.
